.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Prettier plugin for Idea seems to require that this file be named .prettierrc to respect it

Reply:
Ah ok. I was about to quote the docs saying that it can have a `js` or `json` extension, but you're saying the plugin specifically. I will change this to appease the plugin.